fieldservice/models/fsm_order.py

```diff
@@ -377,6 +377,8 @@ def onchange_scheduled_date_end(self):
     @api.onchange("scheduled_duration")
     def onchange_scheduled_duration(self):
         if self.scheduled_duration and self.scheduled_date_start:
+            if self.scheduled_duration < 0:
+                raise ValidationError(_("Scheduled Duration Should be Postive Value"))
```

Comment:
or = 0 ?

Reply:
The default value is zero so no need to check = into it.